.travis.yml

```diff
@@ -4,8 +4,8 @@ python:
   - 3.5
   - "nightly"
 install:
-  - pip install -r requirements-tests.txt
+  - pip install --upgrade -r requirements-tests.txt
 script:
-  - coverage run --source xonsh -m py.test -q
+  - py.test --flake8 --cov=. --cov-report=term
```

Comment:
I don't know if we shouldn't be harcoding exactly what we don't care about? maybe we want to have test for amalgamation that is outside, maybe tests for other examples and other things as well? if not I can totally change.